pontoon/base/fluent.py

```diff
@@ -12,6 +12,14 @@ def get_default_variant(variants):
             return variant
 
 
+def get_variant_key(variant):
+    """Return the key of the variant as represented in the syntax."""
+    if isinstance(variant.key, ast.NumberLiteral):
+        return variant.key.value
+
+    return variant.key.name
```

Comment:
This is perfectly fine and doesn't need to be changed, just thought I'd note that I'd find this style much clearer:

```suggestion
    key = variant.key
    return key.value if isinstance(key, ast.NumberLiteral) else key.name
```